cli/integration_tests/turbo_help.t

```diff
@@ -13,12 +13,8 @@ Test help flag
     completion  Generate the autocompletion script for the specified shell
     daemon      Runs the Turborepo background daemon
     help        Help about any command
-    link        Link your local directory to a Vercel organization and enable remote caching.
```

Comment:
Maybe we should output an error if people are hitting the Go help? Because that shouldn't happen and it might confuse people to see a help screen that's incorrect.